tests/test.test.js

```diff
@@ -0,0 +1,235 @@
+const request = require("supertest");
+const app = require("../app");
+const { db } = require("../models");
+const { registerSchema, loginSchema } = require("../schema");
+
+let email = `john${Date.now()}@example.com`;
+let username = `johnuser${Date.now()}`;
+let password = "StrongPassword123";
+
+beforeAll(async () => {
+  await db.User.destroy({ where: { email } });
+  await db.sequelize.authenticate();
+  await db.sequelize.sync({ force: false });
```

Comment:
I think the destroy should come after syncing the db.
```suggestion
  await db.sequelize.authenticate();
  await db.sequelize.sync({ force: false });
  await db.User.destroy({ where: { email } });
```

Reply:
Fixed and added more test for the max length.